### ISSUE TITLE
Incorrect brackets parsing in links / Некорректное преобразование скобок в ссылках

### ISSUE THREAD

**Issue by kabalin** [PastVu/pastvu#583]
Сообщил [konl](https://pastvu.com/u/konl)·[Вчера в 11:10](https://pastvu.com/news/155?hl=comment-2821866). 
Если ссылка содержит завершающую скобку, она будет рассматриваться как знак препинания, таким образом ссылка будет искажена.

В качестве решения - при преобразовании html encoded ссылки, не декодировать знаки препинания (в частности те, которые парсер считает за знак препинания).

Regression caused by #538

**Comment by darkblaze69** [comment]
Is there workaround for this?

**Comment by kabalin** [comment]
> Is there workaround for this?

Yes, replace last preposition character in URL with its encoded version.

Say, the link is `https://ru.wikipedia.org/wiki/Памятник_Александру_II_(Белый_Ключ)`.
You need to replace the trailing `)` with `%29`, so the new link will be:
`https://ru.wikipedia.org/wiki/Памятник_Александру_II_(Белый_Ключ%29`. This will save it correctly in comment, but if you decide to edit it, you will have to repeat the process.
